.vscode/settings.json

```diff
@@ -17,5 +17,9 @@
       "xstddef": "c",
       "type_traits": "c",
       "utility": "c"
+    },
+    "[markdown]": {
+        "editor.trimAutoWhitespace": false,
+        "files.trimTrailingWhitespace": false
```

Comment:
```suggestion
```

Reply:
How come? Without this, VSCode breaks the formatting @fauxpark relies on for docs?

Reply:
Will raise a separate PR for this change.